apps/src/netsim/NetSimLogPacket.html.ejs

```diff
@@ -56,68 +99,77 @@
   <%
   }
  %>
-<table>
-  <thead>
-    <tr>
-      <th nowrap class="<%= PacketUIColumnType.ENCODING_LABEL %>"></th>
-      <% if (showToAddress) { %>
-        <th nowrap class="<%= PacketUIColumnType.TO_ADDRESS %>"><%= i18n.to() %></th>
-      <% } %>
-      <% if (showFromAddress) { %>
-        <th nowrap class="<%= PacketUIColumnType.FROM_ADDRESS %>"><%= i18n.from() %></th>
-      <% } %>
-      <% if (showPacketInfo) { %>
-        <th nowrap class="<%= PacketUIColumnType.PACKET_INFO %>"><%= i18n.packet() %></th>
-      <% } %>
-      <th nowrap class="<%= PacketUIColumnType.MESSAGE %>"><%= i18n.message() %></th>
-    </tr>
-  </thead>
-  <tbody>
   <%
     var toAddress = showToAddress ? packet.getHeaderAsBinary(Packet.HeaderType.TO_ADDRESS) : '';
     var fromAddress = showFromAddress ? packet.getHeaderAsBinary(Packet.HeaderType.FROM_ADDRESS) : '';
     var packetIndex = showPacketInfo ? packet.getHeaderAsBinary(Packet.HeaderType.PACKET_INDEX) : '';
     var packetCount = showPacketInfo ? packet.getHeaderAsBinary(Packet.HeaderType.PACKET_COUNT) : '';
     var message = packet.getBodyAsBinary();
+  %>
+  <% if (isMinimized) { %>
+      <% packetControls(); %>
+      <div class="single-line-with-ellipsis user-data"><%= getOneLinePacketSummary() %></div>
+  <% } else { %>
+    <table>
+      <thead>
+        <tr>
+          <th nowrap class="<%= PacketUIColumnType.ENCODING_LABEL %>"></th>
+          <% if (showToAddress) { %>
+            <th nowrap class="<%= PacketUIColumnType.TO_ADDRESS %>"><%= i18n.to() %></th>
+          <% } %>
+          <% if (showFromAddress) { %>
+            <th nowrap class="<%= PacketUIColumnType.FROM_ADDRESS %>"><%= i18n.from() %></th>
+          <% } %>
+          <% if (showPacketInfo) { %>
+            <th nowrap class="<%= PacketUIColumnType.PACKET_INFO %>"><%= i18n.packet() %></th>
+          <% } %>
+          <th class="<%= PacketUIColumnType.MESSAGE %>">
+            <% packetControls(); %>
+            <%= i18n.message() %>
+          </th>
+        </tr>
+      </thead>
+      <tbody>
+      <%
+        logRow(EncodingType.ASCII,
+            binaryToInt(toAddress),
+            binaryToInt(fromAddress),
+            i18n.xOfYPackets({
+              x: binaryToInt(packetIndex),
+              y: binaryToInt(packetCount)
+            }),
+            binaryToAscii(message, chunkSize));
+
+        logRow(EncodingType.DECIMAL,
+            binaryToInt(toAddress),
+            binaryToInt(fromAddress),
+            i18n.xOfYPackets({
+              x: binaryToInt(packetIndex),
+              y: binaryToInt(packetCount)
+            }),
+            alignDecimal(binaryToDecimal(message, chunkSize)));
+
+        logRow(EncodingType.HEXADECIMAL,
```

Comment:
confused -- how are these log row calls (for encodings which are not represented in the incoming packet) not showing up in your ani gif?